src/main/java/no/nav/pto/veilarbportefolje/aktiviteter/AktivitetService.java

```diff
@@ -63,7 +67,27 @@ public void deaktiverUtgatteUtdanningsAktivteter() {
         );
     }
 
-    public List<Moteplan> hentMoteplan(VeilederId veilederIdent, EnhetId enhet) {
-        return aktiviteterRepositoryV2.hentFremtidigeMoter(veilederIdent, enhet);
+    public List<Moteplan> hentMoteplan(VeilederId veilederIdent, EnhetId enhet, boolean tilgangTilKode6, boolean tilgangTilKode7, boolean tilgangTilEgenAnsatt) {
+        List<Moteplan> moteplans = aktiviteterRepositoryV2.hentFremtidigeMoter(veilederIdent, enhet);
```

Comment:
Can you somehow pass `DISKRESJONSKODE` in `Moteplan` as attribute in `Motedeltaker` or as seperate field in Moteplan...and then `sensurerMoteplaner` will decorate those meetings further. This suggestion aims to remove call of `hentBrukereSomSkalSensureres` in `sensurerMoteplaner` function.